packages/remme-node-management/src/interface.ts

```diff
@@ -0,0 +1,8 @@
+import { IBaseTransactionResponse } from "remme-transaction-service";
+import { ISetBetPayload } from "remme-protobuf";
+
+export interface IRemmeNodeManagement {
+    open(initialStake: number): Promise<IBaseTransactionResponse>;
+    close(): Promise<IBaseTransactionResponse>;
+    setBet(payload: ISetBetPayload): Promise<IBaseTransactionResponse>;
```

Comment:
provide also [get_node_info, get_node_account and get_node_config](https://remmeio.atlassian.net/wiki/spaces/WikiREMME/pages/292814862/RPC+API+specification) methods